applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py

```diff
@@ -49,6 +50,7 @@ def AssambleTestSuites():
     smallSuite.addTest(TNavierStokesWallConditionTest('test_execution'))
     smallSuite.addTest(BuoyancyTest('testEulerian'))
     smallSuite.addTest(BuoyancyTest('testThermalExpansionCoefficient'))
+    smallSuite.addTest(DarcyChannelTest('testDarcyDensity'))
```

Comment:
are u sure it should be in the "smallSuite"? it is not really small i think

Reply:
Well no, I'm not sure. It is smaller than several of the existing "small" tests, though...
I'd say we leave this discussion for now and move it were appropriate within #872.